packages/@aws-cdk/app-delivery/test/test.pipeline-deploy-stack-action.ts

```diff
@@ -188,7 +188,7 @@ export = nodeunit.testCase({
       adminPermissions: false,
       role
     });
-    test.deepEqual(role.id, deployAction.role.id);
+    test.deepEqual(role.id, (deployAction.role as iam.Role).id);
```

Comment:
This looks fishy. Maybe we should just check equality between the objects?

Reply:
Done.